api/src/test/java/org/openmrs/api/EncounterServiceTest.java

```diff
@@ -3235,4 +3255,89 @@ public void saveEncounter_shouldCascadeSaveToContainedAllergies() {
 		assertTrue(allergies.contains(allergy));
 		assertEquals(NAMESPACE + "^" + FORMFIELD_PATH, allergies.iterator().next().getFormNamespaceAndPath());
 	}
+
+	@Test
+	public void getEncountersByVisitsAndPatient_shouldReturnCorrectEncounters() {
+		Patient patient = Context.getPatientService().getPatient(3);
+
+		List<Encounter> encounters = Context.getEncounterService()
+			.getEncountersByVisitsAndPatient(patient, false, StringUtils.EMPTY, 0, 5);
```

Comment:
Is there a reason why you did not just use `null` instead of `StringUtils.EMPTY`?

Reply:
No specific reason, updated to use null